components/raftstore/src/store/fsm/peer.rs

```diff
@@ -873,33 +873,45 @@ where
         self.register_raft_base_tick();
     }
 
+    fn finish_unsafe_recovery_wait_apply(&mut self) {
+        if self
+            .fsm
+            .unsafe_recovery_wait_apply_counter
+            .as_ref()
+            .unwrap()
```

Comment:
it may panic here

Reply:
Are you worrying about the "destroy peer" case may panic here, or panic due to race condition?

Reply:
about the "destroy_peer", now it's sovled